build-openjdk8.sh

```diff
@@ -43,7 +81,7 @@ build() {
 
   for debug in release slowdebug; do
     bash configure \
-       --with-boot-jdk="/usr/lib/jvm/java-1.7.0-openjdk.x86_64" \
+       --with-boot-jdk="$(boot_jdk)" \
```

Comment:
I realize the old boot jdk path wasn't super great to begin with (`/usr/lib/jvm/java-1.7.0-openjdk.x86_64`). Sorry about that. Please use, `--with-boot-jdk="/usr/lib/jvm/java"` which should exist as soon as the `java-1.7.0-openjdk-devel` package is installed. This removes the need to do a switch on the platform architecture in order to figure out the boot JDK path.

On an aarch64 system:
```
# /usr/lib/jvm/java/bin/java -version
java version "1.7.0_221"
OpenJDK Runtime Environment (rhel-2.6.18.0.el7_6-aarch64 u221-b02)
OpenJDK 64-Bit Server VM (build 24.221-b02, mixed mode)
```
On an x86_64 system:
```
# /usr/lib/jvm/java/bin/java -version
java version "1.7.0_261"
OpenJDK Runtime Environment (rhel-2.6.22.1.el6_10-x86_64 u261-b02)
OpenJDK 64-Bit Server VM (build 24.261-b02, mixed mode)
```